### PR TITLE
fix: migrate media files from fs-blob-store to safe-fs-blob-store 

### PR DESCRIPTION
**Pull request by karissa**
This little PR migrates media blob stores from fs-blob-store to safe-fs-blob-store

If media is not migrated, then people using older versions of Mapeo Mobile (e.g., the version ECA has) who upgrade to the newest version will not be able to see or sync the images they've taken previously.

### CODE REVIEW

**Comment by karissa** [comment]
@noffle for some reason there seems to be a race condition with writing and reading the files from the blob stores, can you help me figure out what is going on there? The tests don't pass consistently

**Comment by karissa** [comment]
screenshot from 2018-10-05 17-13-23
@noffle after running tests, it errors saying file doesn't exist, but when i ls the dir, I see the file there.

File: src/rnnodeapp/index.js [hunk 1]
```diff
@@ -47,13 +49,20 @@ mkdirp.sync(MEDIA_PATH);
 mkdirp.sync(STATIC_PATH);
 console.log('2: dirs created');
 
+// Migrate media from old fs-blob-store, if needed
+migrateMedia(MEDIA_PATH, function (err) {
+  if (err) console.error(err);
```
Comment (noffle) [inline src/rnnodeapp/index.js]: Won't this potentially start the app before migration is complete?

File: src/rnnodeapp/migrateMedia.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      })
+    })
+    parallel(tasks, cb)
+  })
+
+  function migrateFile (key, cb) {
+    var filename = path.join(mediaPath, key)
+    fs.stat(filename, function (err, stat) {
+      if (err) return cb(err)
+      if (stat.isFile()) {
```
Comment (noffle) [inline src/rnnodeapp/migrateMedia.js]: `cb` is never called if `isFile()` isn't truthy. will `parallel` hang?

Comment (karissa) [reply src/rnnodeapp/migrateMedia.js]: addressed! nice catch

**Comment by noffle** [comment]
Awesome! All that's left is testing it on a real device /w legacy data and then seeing if the new version of the app can see media. (*groan*)

cc @gmaclennan

**Pull request merged**